notices.json

```diff
@@ -1,38 +1,4 @@
 [
-  {
-    "id": "use_case_survey",
-    "conditions": {
-      "displayDate": "<= 2022-08-01T00:00:00Z"
-    },
-    "localizedMessages": {
-      "en": {
-        "title": "We want your feedback!",
-        "description": "We are looking for feedback on how we are doing in supporting your team's collaboration throughout different phases of the DevOps lifecycle. Please help us with a quick 5 minutes survey for a chance to win a set of AirPod Pros.",
-        "image": "https://raw.githubusercontent.com/mattermost/notices/master/images/Feedback.png",
```

Comment:
Should this image be removed from the repo or this is reusable and will remain as is?

Same question for:
- https://raw.githubusercontent.com/mattermost/notices/release/images/mattermost-cloud-freemium.png
- https://raw.githubusercontent.com/mattermost/notices/master/images/v6.0.gif
- https://raw.githubusercontent.com/mattermost/notices/master/images/Boards_v6.2.gif

Reply:
Images are reusable so no need to remove them